aws/provider_test.go

```diff
@@ -981,6 +982,55 @@ func testAccPreCheckSkipError(err error) bool {
 	return false
 }
 
+// testSweepOrchestrator is a convenience function that calls testSweepOrchestratorWithData
+//nolint
+func testSweepOrchestrator(jobs []string, r *schema.Resource, region string) error {
+	return testSweepOrchestratorWithData(jobs, r, r.Data(nil), region)
+}
+
+// testSweepOrchestratorWithData launches goroutines to perform sweeping work
+func testSweepOrchestratorWithData(jobs []string, r *schema.Resource, d *schema.ResourceData, region string) error {
+	var wg sync.WaitGroup
+	wgDone := make(chan bool)
```

Comment:
`wgDone` doesn't appear to be written to or read from

Reply:
It is now.

Reply:
Sorry I'm being completely dense, where is this being used?